pilot/pkg/util/network/ip.go

```diff
@@ -193,6 +202,22 @@ func AllIPv4(ipAddrs []string) bool {
 	return true
 }
 
+// CheckIPFamilyTypeForFirstIPs checks the ip family type for the first ip addresses
+func CheckIPFamilyTypeForFirstIPs(ipAddrs []string) IPFamilyType {
+	if len(ipAddrs) == 0 {
+		return UNKNOWN
+	}
```

Comment:
if this happens it should return an error that should be propagated up

Reply:
Hi @jacob-delgado okay, done.